pybossa/uploader/local.py

```diff
@@ -70,5 +70,13 @@ def delete_file(self, name, container):
             path = os.path.join(self.upload_folder, container, name)
             os.remove(path)
             return True
-        except:
+        except Exception:
             return False
+
+    def file_exists(self, name, container):
+        """Check if a file exists in a container"""
+        try:
+            path = os.path.join(self.upload_folder, container, name)
+            return os.path.isfile(path)
+        except Exception:
```

Comment:
This will hide errors. We should only check for exceptions returned by os.path module ;-)